### PR TITLE
adding alliance changes

### PR DESCRIPTION
**Pull request by jhdalek55**
made change to Navbar, and to Alliance, Index and overview page to remove Alliance mentions and add in the current legal name for JDF Uptane.

Changes still need to be made on the Timeline, People, Participate, and Governance 

Note: I am setting up a separate pull request for the cover page to hold the versions.

### CODE REVIEW

**Comment by jhdalek55** [comment]
@JustinCappos @patrickvacek @trishankatdatadog  can one or two of you check this. This PR has been open for over a month. It's simply cleaning up references to the Alliance.

File: alliance.md [hunk 1]
```diff
@@ -4,22 +4,21 @@ css_id: alliance
...
+# Uptane: A JDF/Linux Foundation Project
 
 The Uptane Alliance was formally established in July of 2018 for the express
 purpose of standardizing an open and secure design for over-the-air updates
 for automobiles.  In practice, this process has involved improving and
 standardizing the Uptane open-source software. Formed under the umbrella of
-IEEE's [International Standards and Technology Organization](https://ieee-isto.org/) and now 
-a [Linux Foundation Joint Development Foundation](http://www.jointdevelopment.org/) alliance, the
-Alliance is a nonprofit consortium of companies, non-profit groups, and government agencies.
+IEEE's [International Standards and Technology Organization](https://ieee-isto.org/), the legal name of the initiative is now the Joint Development Foundation
```
Comment (trishankatdatadog) [inline alliance.md]: Sorry, I'm confused. It was formed under IEEE/ISTO, but now only the legal name has changed, or is it now officially under JDF/LF?

File: alliance.md [hunk 2]
```diff
@@ -4,22 +4,21 @@ css_id: alliance
...
+Projects, LLC, Uptane Series, and it is hosted by the
```
Comment (trishankatdatadog) [inline alliance.md]: The legal name is not entirely clear to me. Is it "JDF Project, LLC, Uptane Series" or "JDF Projects, LLC Uptane"?

File: alliance.md [hunk 3]
```diff
@@ -4,22 +4,21 @@ css_id: alliance
...
 purpose of standardizing an open and secure design for over-the-air updates
 for automobiles.  In practice, this process has involved improving and
 standardizing the Uptane open-source software. Formed under the umbrella of
-IEEE's [International Standards and Technology Organization](https://ieee-isto.org/) and now 
-a [Linux Foundation Joint Development Foundation](http://www.jointdevelopment.org/) alliance, the
-Alliance is a nonprofit consortium of companies, non-profit groups, and government agencies.
+IEEE's [International Standards and Technology Organization](https://ieee-isto.org/), the legal name of the initiative is now the Joint Development Foundation
+Projects, LLC, Uptane Series, and it is hosted by the  
+[Joint Development Foundation](http://www.jointdevelopment.org/) of the
+[Linux Foundation](http://www.linuxfoundation.org/). JDF Projects, LLC Uptane is a nonprofit consortium of professionals from industry,companies, non-profit groups, and government agencies.
```
Comment (trishankatdatadog) [inline alliance.md]: See comment above about the legal name.

File: alliance.md (hunk 1 above)
Comment (jhdalek55) [reply alliance.md]: It is officially under JDF/LF. Let me clarify the wording.

File: alliance.md (hunk 2 above)
Comment (jhdalek55) [reply alliance.md]: I'm going to check and change. Good catches.

File: alliance.md (hunk 1 above)
Comment (jhdalek55) [reply alliance.md]: Done

File: alliance.md (hunk 2 above)
Comment (jhdalek55) [reply alliance.md]: Fixed

**Comment by jhdalek55** [comment]
Thank you

**Comment by trishankatdatadog** [comment]
You are most welcome, Lois, thank you!

**Pull request merged**